spec/presenter_spec.rb

```diff
@@ -5,16 +5,16 @@
   describe "#display_board" do
     it "formats the board" do
       presenter = Presenter.new
-      board = Array.new(9)
-      board[4] = "X"
-      blank_space = " "
+      board = Board.new
+      board.place_token(5, "X")
+      blank_string = " "
       expected_board = <<~BOARD
 
-         #{board[0] || blank_space} | #{board[1] || blank_space} | #{board[2] || blank_space}
+           |   | #{blank_string}
         ---+---+---
-         #{board[3] || blank_space} | #{board[4] || blank_space} | #{board[5] || blank_space}
+           | X | #{blank_string}
         ---+---+---
-         #{board[6] || blank_space} | #{board[7] || blank_space} | #{board[8] || blank_space}
+           |   | #{blank_string}
```

Comment:
Nice way to signal trailing whitespace 👍

Reply:
neat!!